fml/log_level.h

```diff
@@ -16,6 +16,11 @@ constexpr LogSeverity LOG_ERROR = 2;
 constexpr LogSeverity LOG_FATAL = 3;
 constexpr LogSeverity LOG_NUM_SEVERITIES = 4;
 
+// It seems that one of the Windows header files defines ERROR as 0.
```

Comment:
Can you elaborate here?  As-is, this comment isn't very clear.  Do you mean that because ERROR is 0, some macro somewhere does `LOG_ ## ERROR` to produce a `LOG_0` token, and therefore we must alias `LOG_0` to `LOG_ERROR`?

Reply:
Yeah, exactly that.

Reply:
Ah, ok, I just saw #6675.  How about adjusting the comment to add something like "Therefore we must alias LOG_0 to LOG_ERROR so that FML_LOG(ERROR) works as expected." (Or just reference #6675.)

Reply:
I updated the message.